libs/kml/serdes_geojson.cpp

```diff
@@ -78,22 +79,62 @@ bool GeojsonParser::Parse(std::string_view jsonContent)
   {
     if (auto const * point = std::get_if<GeoJsonGeometryPoint>(&feature.geometry))
     {
-      double const longitude = point->coordinates.at(0);
-      double const latitude = point->coordinates.at(1);
+      double longitude = point->coordinates.at(0);
```

Comment:
Here it is not a nit, but a required change, because it runs in a loop.
```suggestion
      double longitude = point->coordinates[0];
```